test/Passes/tile-and-fuse-default.mlir

```diff
@@ -0,0 +1,238 @@
+// RUN: tpp-opt %s -split-input-file -tile-consumer-and-fuse-producers -cse | FileCheck %s
+
+// CHECK-NOT: scf.for
```

Comment:
Nit: better to match and put the CHECK line after the function like the others, to make it easier to know which op it applies to.

Also, add a check for the `func.func`, to make sure we're in the right place. Since this is a split-file, we may add tests before and the check will always pass.